include/hipSYCL/sycl/buffer.hpp

```diff
@@ -674,12 +674,12 @@ class buffer : public detail::property_carrying_object
   }
 
   template<typename... Args>
-  auto get_access(Args... args) {
+  auto get_access(Args&&... args) {
     return accessor{*this, args...};
   }
 
   template<typename... Args>
-  auto get_host_access(Args... args) {
+  auto get_host_access(Args&&... args) {
```

Comment:
ain't this a forward reference -> you'd ideally do `std::forward<Args>(args)...`?

Reply:
You are right, using `std::forward` is better. Fixed :)